index.bs

```diff
@@ -3530,6 +3530,26 @@ operation. Likewise, clients can choose to produce a [=client extension output=]
 understand by encoding the [=authenticator extension output=] value into JSON, provided that the CBOR output uses only types
 present in JSON.
 
+When clients choose to pass through extensions they do not recognize,
+the JavaScript values in the [=client extension input=] values
+are converted to [=CBOR=] values in the [=authenticator extension input=] values.
+When the JavaScript input value is an [=%ArrayBuffer%=], it is converted to a [=CBOR=] byte array.
+When the JavaScript input value is a non-integer number, it is converted to a 64-bit CBOR floating point number.
+Otherwise, when the JavaScript input type corresponds to a JSON type, the conversion is done
+using the rules defined in Section 4.2 of [[RFC7049]] (Converting from JSON to CBOR),
+but operating on JavaScript inputs rather than JSON inputs.
```

Comment:
s/ JavaScript inputs rather than JSON inputs / inputs of JavaScript type values rather than JSON-encoded inputs /   ...?

should perhaps generally reference https://tc39.github.io/ecma262/#sec-ecmascript-data-types-and-values ?